include/libopencm3/stm32/dsi.h

```diff
@@ -0,0 +1,25 @@
+/* This provides unification of code over STM32F subfamilies */
```

Comment:
I know it's all over the place, but we should try and remember the L devices too one day :)  there's also T and there was W ;)